### PR TITLE
[PWD-2486] - event detail page

### PR DESCRIPTION
**Pull request by ansorudin**
### What type of PR is this? (_check all applicable_)

- [X] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

File: src/views/event-detail/components/SpeakerSection.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export const SpeakerSection: FC<SpeakerSectionProps> = ({ speakers }) => {
+  return (
+    <EventDetailWrapper title="Tentang Instruktur" isBorder={false}>
+      <Stack spacing="1rem" mt="1rem">
+        {Children.toArray(
+          speakers?.map((speaker) => (
+            <SpeakerCard key={speaker.name} {...speaker} showDescription />
+          ))
+        )}
```
Comment (shadiqfauzi) [inline src/views/event-detail/components/SpeakerSection.tsx:17]:
```suggestion
        {Children.toArray(
          speakers?.map((speaker) => (
            <SpeakerCard {...speaker} showDescription />
          ))
        )}
```
kalau udah pake Children.toArray gak perlu pasang key lagi bukan?

File: src/views/event-detail/components/ContentSection.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+              ))
+            )}
+          </OrderedList>
+        </ContentDetailsWrapper>
+        <ContentDetailsWrapper
+          title="Siapa saja yang dapat mengikuti acara ini?"
+          isBorder={!!workshopDetail.notes}
+        >
+          <UnorderedList>
+            {parseAudience.map((audience, idx) => (
```
Comment (Vesals) [inline src/views/event-detail/components/ContentSection.tsx:66]: ketinggalan nih belum pake Children.toArray()

Comment (shadiqfauzi) [reply src/views/event-detail/components/ContentSection.tsx:66]: yup kalau gaada id yg bisa dijadiin key pake Children.toArray() aja biar ga perlu passing key index

File: src/features/program/types.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  endTime: string
+  workshopDetail: WorkshopDetail
+  speakers: Speaker[]
+  branch: Branch
+  programHeader: ProgramHeader
+}
+
+export interface ProgramEventDetailResponse {
+  result: Program
+}
```
Comment (FahemHakikiKhaya) [inline src/features/program/types.ts:39]: ini bisa nunggu event page naik, udh gw rapihin sama pisahin2 typenya

Comment (ansorudin) [reply src/features/program/types.ts:39]: wokeeehh

File: src/views/event-detail/components/ContentSection.tsx (hunk 2 above)
Comment (ansorudin) [reply src/views/event-detail/components/ContentSection.tsx:66]: ahh iya, thanks

**Review by kuntiarso — APPROVED** [review]

**Review by haikal1223 — DISMISSED** [review]

**Review by Vesals — APPROVED** [review]

**Review by katherinedavenia — DISMISSED** [review]

File: src/views/event-detail/components/ContentDetailWrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+interface ContentDetailsWrapperProps extends PropsWithChildren {
+  title: string
+  isBorder?: boolean
+}
+
+export const ContentDetailsWrapper: FC<ContentDetailsWrapperProps> = ({
+  children,
+  title,
+  isBorder = true
```
Comment (katherinedavenia) [inline src/views/event-detail/components/ContentDetailWrapper.tsx:12]:
```suggestion
  showBorder = true
```

**Review by shadiqfauzi — APPROVED** [review]
aku sih yes

**Review by Vesals — APPROVED** [review]

**Pull request merged**